vignettes/wrappers.Rmd

```diff
@@ -25,7 +25,7 @@ Therefore, you'd prefer to use `<<` and `>>` as the opening and closing delimite
 Spoiler alert: here's the correct way to write such a wrapper:
 
 ```{r}
-myglue <- function(..., .envir = parent.frame()) {
+my_glue <- function(..., .envir = parent.frame()) {
```

Comment:
Might be worth a footnote pointing out that this is the same pattern you use in `abort()`/`cli_abort()` wrappers? And in `defer()`?

Reply:
I was thinking the same! I added a sentence in the intro. I think it's worthy to point this out.